### ISSUE TITLE
Moving sidebar to the right overlays scroll bar 

### ISSUE THREAD

**Issue by sana-ajani** [microsoft/vscode#129609]
Version: 1.59.0-insider
Commit: c3b4df8f5d9c14aa4cfddb587231d0d52e72e5a2
Date: 2021-07-27T15:10:39.610Z
Electron: 13.1.7
Chrome: 91.0.4472.124
Node.js: 14.16.0
V8: 9.1.269.36-electron.0
OS: Darwin x64 18.7.0

Steps to Reproduce:
1. Open any repo in VS Code and open a preview of the readme
2. From menu, click on View > Appearance > Move sidebar right
3. Notice the scroll bar for the readme is overlayed onto the sidebar

![image](https://user-images.githubusercontent.com/9721254/127216639-0c1287b9-b1f9-4d12-848d-fc3d9d900521.png)

**Comment by jeanp413** [comment]
duplicate of #129120

**Comment by abdlrhmanshehatamoussa** [comment]
Hi Guys,

I'm ready to help,

If any one could give a little push on where exactly can find the relevant code and any other recommendations, I'm ready to gooo.

Best of luck !

**Comment by rebornix** [comment]
duplicate of https://github.com/microsoft/vscode/issues/129120 .

**Comment by mjbvz** [comment]
Yes same root cause as #129120 but keeping open so I can test and verify it separately
